src/themes/default/assets/css/bootstrap.css

```diff
@@ -10428,4 +10428,9 @@ a.profilelink:link {
 
 #reflist p {
     word-wrap: break-word;
+}
+
+.feature-article-image {
```

Comment:
Just a Q,
Given that you added this class to `src/static/default/css/bootstrap.css` why do we also need it here?

Reply:
build_assets can combine css/scss from each theme to make overrides for each journal before copying it over to static

Reply:
Ah I see, `src/static/default/css/bootstrap.css` is autogenerated, perhaps it would be best to not add it to version control